### ISSUE TITLE
Accessibility 2026 - Multiple

### ISSUE THREAD

**Issue by MikeDelue** [ua-snap/usda-dash#45]
- Page language is missing or invalid. Identify the document language using the <html lang> attribute with a valid value (e.g., <html lang="en">). Ensure that all lang attribute values are valid.
- Missing alt text for: SNAP logo in header - SNAP, UAF and AK CASC logos in footer. (Also this CASC footer is ancient but that's not an accessibility concern.)
- 2 missing form labels and 3 orphan form labels. These corrections should cancel one another out:

<img width="1604" height="342" alt="Image" src="https://github.com/user-attachments/assets/ccb63460-bf7d-4394-8af8-67b9bd03e3da" />

- Heading levels skipped twice. We should find a way to not skip heading levels:

<img width="876" height="204" alt="Image" src="https://github.com/user-attachments/assets/5b411fea-9878-4fd4-a5c7-81de8b7fc97a" />

<img width="730" height="186" alt="Image" src="https://github.com/user-attachments/assets/74309766-28b0-415d-99b4-ec6e7fd5cbd1" />

- Consider removing this link to the Peonie PDF. PDFs are likely to be archived as a result of these new accessibility guidelines

<img width="1039" height="309" alt="Image" src="https://github.com/user-attachments/assets/2a81fc8d-4ac0-428e-9243-e64467927090" />

- Make the whole sentence "Learn more about specific crops here" the link, instead of just "here" for link clarity reasons:

<img width="330" height="94" alt="Image" src="https://github.com/user-attachments/assets/92a88f37-55a3-4bef-a452-9c062a698dad" />

**Comment by MikeDelue** [comment]
Hardiness Zones tab:

- All four maps are missing alt text.
- Skipped header levels for "Download Maps", "About Hardiness", and "Where do these numbers come from"
- Peonie PDF link repeats.

Annual Minimums tab:
- All of the same fixes from the Growing Season tab

<img width="634" height="602" alt="Image" src="https://github.com/user-attachments/assets/3e4eccd2-cf23-48eb-9faa-98e83b41c3e7" />

GDD tab:
- Many of the same fixes from the Growing Season tab

<img width="1169" height="539" alt="Image" src="https://github.com/user-attachments/assets/45907bcf-3624-41e6-994c-ecf081a097e3" />
